internal/services/automation/automation_variable_datetime_resource_test.go

```diff
@@ -135,3 +151,31 @@ resource "azurerm_automation_variable_datetime" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger)
 }
+
+func (AutomationVariableDateTimeResource) encrypted(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-auto-%d"
+  location = "%s"
+}
+
+resource "azurerm_automation_account" "test" {
+  name                = "acctestAutoAcct-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku_name            = "Basic"
+}
+
+resource "azurerm_automation_variable_datetime" "test" {
+  name                    = "acctestAutoVar-%d"
+  resource_group_name     = azurerm_resource_group.test.name
+  automation_account_name = azurerm_automation_account.test.name
+  value                   = "2019-04-24T21:40:54.074Z"
+  encrypted               = true
```

Comment:
instead of creating an entire new test for this can we just add it to the complete one? thanks

Reply:
actually since the Encrypted/Unencrypted use-cases are different in the API can we split this out into a separate test to ensure we handle the Encrypted workflow?

Reply:
@tombuildsstuff I've updated the PR. please bother to check again. Thanks!